roles/ceph-rgw-loadbalancer/defaults/main.yml

```diff
@@ -5,6 +5,11 @@
 # GENERAL #
 ###########
 
+keepalived_force_update:
```

Comment:
This shouldn't be necessary. If someone sets the version different from default, we can assume they want that sym-linked.

Reply:
removed